collectors/ebpf.plugin/ebpf_socket.c

```diff
@@ -320,8 +321,8 @@ static void ebpf_socket_send_data(ebpf_module_t *em)
     write_count_chart(NETDATA_UDP_FUNCTION_COUNT, NETDATA_EBPF_IP_FAMILY,
                       &socket_publish_aggregated[NETDATA_IDX_UDP_RECVBUF],2);
     write_io_chart(NETDATA_UDP_FUNCTION_BITS, NETDATA_EBPF_IP_FAMILY,
-                   socket_id_names[3],(long long)common_udp.write * 8/BITS_IN_A_KILOBIT,
-                   socket_id_names[4], (long long)common_udp.read * 8/BITS_IN_A_KILOBIT);
+                   socket_id_names[3], (long long)common_udp.read * 8/BITS_IN_A_KILOBIT,
+                   socket_id_names[4], (long long)common_udp.write * 8/BITS_IN_A_KILOBIT);
```

Comment:
just a comment:

the function "expects" write,read

https://github.com/netdata/netdata/blob/bc3d8c99066fd80b8f99f10cba8b87ed7404c025/collectors/ebpf.plugin/ebpf.c#L386-L394

And it is used in several places

https://github.com/netdata/netdata/search?q=write_io_chart

Reply:
I am doing a big work to reorganize the dashboard and I will have to address all threads that we have merged, I will do necessary changes when I work again with socket. :handshake: